chart/compass/templates/director-api-test.yaml

```
@@ -13,6 +13,8 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "true"
+      labels:
+        app: {{ .Chart.Name }}-tests-director-api
```

Please move the `tests` suffix at the end of the label to be consistent
```suggestion
        app: {{ .Chart.Name }}-director-api-tests
```